kornia/geometry/nerf/nerfmm/models/intrinsics.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import torch
+import torch.nn as nn
+
+
+class LearnFocal(nn.Module):
```

Comment:
similar to `PinholeCamera` this could be a class `PinholeCameraIntrinsics` with learnable parameters.